src/ric/spriteparts.c

```diff
@@ -0,0 +1,1140 @@
+
+int D_80153968[] = {0xFFF080A0, 0x0000FFE8};
+
+int D_80153970[] = {0xFFF080A1, 0x0000FFE8};
+
+int D_80153978[] = {0xFFF080A2, 0x0000FFE8};
+
+int D_80153980[] = {0xFFF080A3, 0x0000FFE8};
+
+int D_80153988[] = {0xFFF080A4, 0x000BFFE8};
+
+int D_80153990[] = {0xFFF080A5, 0x000CFFE8};
+
+int D_80153998[] = {0xFFF080A6, 0x000DFFE8};
+
+int D_801539A0[] = {0xFFF080A7, 0x0000FFE8};
+
+int D_801539A8[] = {0xFFF080A8, 0x0000FFE8};
+
+int D_801539B0[] = {0xFFF080A9, 0x000FFFE8};
+
+int D_801539B8[] = {0xFFF080AA, 0x000FFFE8};
+
+int D_801539C0[] = {0xFFE880AB, 0x000FFFE8};
+
+int D_801539C8[] = {0xFFF080AC, 0x000FFFD8};
+
+int D_801539D0[] = {0xFFF080AD, 0x000EFFE8};
+
+int D_801539D8[] = {0xFFE880AE, 0x000EFFE8};
+
+int D_801539E0[] = {0xFFED80AF, 0x0011FFE7};
+
+int D_801539E8[] = {0xFFED80AF, 0x0012FFE7};
+
+int D_801539F0[] = {0xFFED80AF, 0x0013FFE7};
+
+int D_801539F8[] = {0xFFF080B0, 0x0010FFE8};
+
+int D_80153A00[] = {0xFFF080B1, 0x0010FFE8};
+
+int D_80153A08[] = {0xFFF080B2, 0x0010FFE8};
+
+int D_80153A10[] = {0xFFF080B3, 0x0010FFE8};
+
+int D_80153A18[] = {0xFFF080B4, 0x0000FFE8};
+
+int D_80153A20[] = {0xFFF080B5, 0x0000FFE8};
+
+int D_80153A28[] = {0xFFF080B6, 0x0000FFE8};
+
+int D_80153A30[] = {0xFFF080B7, 0x0000FFE8};
+
+int D_80153A38[] = {0xFFF080B8, 0x0000FFF8};
+
+int D_80153A40[] = {0xFFF080B9, 0x0000FFF8};
+
+int D_80153A48[] = {0xFFF080BA, 0x00000000};
+
+int D_80153A50[] = {0xFFF080BB, 0x0000FFF8};
+
+int D_80153A58[] = {0xFFF080BC, 0x00000000};
+
+int D_80153A60[] = {0xFFF080BD, 0x0000FFF8};
+
+int D_80153A68[] = {0xFFF080BE, 0x0000FFE8};
+
+int D_80153A70[] = {0xFFF080BF, 0x0000FFE8};
+
+int D_80153A78[] = {0xFFF080C0, 0x0000FFE8};
+
+int D_80153A80[] = {0xFFF080C1, 0x0000FFE8};
+
+int D_80153A88[] = {0xFFF880C2, 0x0000FFE8};
+
+int D_80153A90[] = {0xFFF880C3, 0x0000FFE8};
+
+int D_80153A98[] = {0xFFF080C4, 0x0000FFE8};
+
+int D_80153AF4[];
+int D_80153B10[];
+int D_80153B2C[];
+int D_80153B48[];
+int D_80153B64[];
+int D_80153B80[];
+int D_80153B9C[];
+int D_80153BB8[];
+int D_80153BD4[];
+int D_80153BF0[];
+int D_80153C0C[];
+int D_80153C28[];
+int D_80153C44[];
+int D_80153C60[];
+int D_80153C7C[];
+int D_80153C98[];
+int D_80153CB4[];
+int D_80153CD0[];
+int D_80153CEC[];
+int D_80153D08[];
+
+SpriteParts* D_80153AA0[] = {
+    (SpriteParts*)NULL,       (SpriteParts*)D_80153AF4,
+    (SpriteParts*)D_80153B10, (SpriteParts*)D_80153B2C,
+    (SpriteParts*)D_80153B48, (SpriteParts*)D_80153B64,
+    (SpriteParts*)D_80153B80, (SpriteParts*)D_80153B9C,
+    (SpriteParts*)D_80153BB8, (SpriteParts*)D_80153BD4,
+    (SpriteParts*)D_80153BF0, (SpriteParts*)D_80153C0C,
+    (SpriteParts*)D_80153C28, (SpriteParts*)D_80153C44,
+    (SpriteParts*)D_80153C60, (SpriteParts*)D_80153C7C,
+    (SpriteParts*)D_80153C98, (SpriteParts*)D_80153CB4,
+    (SpriteParts*)D_80153CD0, (SpriteParts*)D_80153CEC,
+    (SpriteParts*)D_80153D08};
+
+int D_80153AF4[] = {0x00300001, 0xFFF5FFF5, 0x00180018, 0x00000000,
```

Comment:
I know you're following the other file but should we not fully extract these? https://github.com/Xeeynamo/sotn-decomp/blob/6898b8b60818dd222d770caf97704635d464164a/include/game.h#L1269

Reply:
Sure, if that's of interest. I think the best option will be to merge this one, and then I will write a script to convert both spriteparts.c files to properly extract the structs, if that works.